src/main/java/com/maxmind/minfraud/response/Email.java

```diff
@@ -1,8 +1,12 @@
 package com.maxmind.minfraud.response;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.maxmind.minfraud.AbstractModel;
 
+import java.time.LocalDate;
+import java.time.ZonedDateTime;
```

Comment:
Are we using this?

Reply:
Woops, I was debating between returning a `ZonedDateTime` and `LocalDate` given that it is really a "zoned date". I'll remove.